RELEASE_NOTES.md

```diff
@@ -1,3 +1,103 @@
+## Cookstyle 5.6
+
+### 12 New Chef Cops
+
+#### ChefDeprecations/WindowsTaskChangeAction
+
+The `ChefDeprecations/WindowsTaskChangeAction` cop detects the `windows_task` resource used with the legacy `:change` action. When the `windows_task` resource was moved into Chef Infra Client 13, the `:change` action was merged into the `:create` action. Mot users set the `:change` action in conjunction with `:create` and `:change` can now be removed.
+
+`Enabled by default`: True
+
+`Autocorrects`: Yes
+
+#### ChefDeprecations/RecipeMetadata
+
+The `ChefDeprecations/RecipeMetadata` cop detects the usage of the `recipe` method in metadata.rb. This method was never used by Chef Infra and it has been deprecated. User facing documentation such as recipe descriptions should instead be placed in the cookbook's README.md file.
```

Comment:
It is interesting to me that we are deprecating something we never used or standardized.

![tenor-206154679](https://user-images.githubusercontent.com/5712253/65050836-49ff5b80-d925-11e9-9882-1dfd49e1be13.gif)

Reply:
We documented it and users used it based on the documentation.